unlock-app/src/components/interface/buttons/homepage/HomepageButton.js

```diff
@@ -0,0 +1,119 @@
+import styled from 'styled-components'
```

Comment:
Maybe more `HomePage` and `HomePageButton`?

Reply:
Homepage is one word!